third_party/nvfuser/csrc/ir_builder.h

```diff
@@ -30,6 +30,10 @@ class TORCH_CUDA_CU_API IrBuilder {
 
     container->registerStmt(IrBuilderPasskey(container), node);
 
+    if (container->activeMultiGroupFusionBuilder()) {
```

Comment:
This could potentially be replaced by
```
if (container->isA<MultiGroupFusion>()) {
```
The problem is that the class needs to be defined first, so I don't know how to organize the files dependancies

Reply:
solved